amethyst_input/src/bindings.rs

```diff
@@ -10,6 +10,28 @@ use super::{Axis, Button};
 /// Used for saving and loading input settings.
 ///
 /// An action can either be a single button or a combination of them.
+///
+/// # Examples
+///
+/// Example Ron config file:
+/// ```ron
+/// (
+///     axes: {
+///         "updown": Emulated(
+///             pos: Key(Up),
+///             neg: Key(Down)
+///         ),
+///         "leftright": Emulated(
+///             pos: Key(Right),
+///             neg: Key(Left)
+///         )
+///     },
+///     actions: {
+///         "fire": [ [Mouse(Left)], [Key(X)] ], // Up to four bindings for one action
+///         "reload": [ [Key(LControl), Key(R)] ] // Combinations of two bindings possible
```

Comment:
Please remove "two" and "four" here.  We optimize for those values as they're very common, but they're not hard limits.  You can have more.